src/mame/nintendo/pin64.cpp

```diff
@@ -4,7 +4,7 @@
 #include "emu.h"
 #include "pin64.h"
 
-#define CAP_NAME "pin64_%d.cap"
+const char* CAP_NAME = "pin64_%d.cap";
```

Comment:
Common pitfall = you made a non-const pointer to a constant string.  Safer to use and array.

Reply:
I already changed it – I made it `static`, too (as well as a few other details), and merged the PR.

Reply:
OK, thanks